src/index.html

```diff
@@ -1,13 +1,418 @@
+                    Max Henry
+                  </a>
+                </h3>
+
+                <div class="testimonial__role">
+                  Chief Design Officer
+                </div>
+
+                <p class="testimonial__about">
+                  When my lovely valley of steam rises and half-day sun is above an impermeable
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contact-us" id="contact-us">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-2">
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Phone
+                </div>
+
+                <a
+                  href="tel:+1 234 5555-55-55"
+                  class="contact-us__info-link"
+                >
+                  +1 234 5555-55-55
+                </a>
+              </div>
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Email
+                </div>
+
+                <a
+                  href="mailto:hello@miami.com"
+                  class="contact-us__info-link"
+                >
+                  hello@miami.com
+                </a>
+              </div>
+            </div>
+            <div class="grid__item grid__item--4-6">
+              <div class="contact-us__info contact-us__field--last">
+                <div class="contact-us__info-title">
+                  Address
+                </div>
+                <a
+                  href="https://goo.gl/maps/6rYGsTDWtWuBGU8w6"
+                  class="contact-us__info-link"
+                >
+                  400 first ave. n.
+                  <br>
+                  suite 700
+                  <br>
+                    Minneapolis, MN
+                  <br>
+                    55401
+                </a>
+              </div>
+            </div>
+            <div class="grid__item grid__item--8-12">
+              <form action="">
+                <input
+                  name ="name"
+                  type="text"
+                  placeholder="Name"
+                  class="contact-us__field form-field"
+                  required
+                >
+                <input
+                  name ="email"
+                  type="email"
+                  placeholder="email@example.com"
+                  class="contact-us__field form-field"
+                  required
+                >
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  class="contact-us__field contact-us__field--last form-field form-field--textarea"
+                  required
+                >
+                </textarea>
```

Comment:
```suggestion
                <textarea
                  name="message"
                  placeholder="Message"
                  class="contact-us__field contact-us__field--last form-field form-field--textarea"
                  required
               ></textarea>
```
Існує баг, коли не відображається плейсхолдер в textarea. Щоб його пофіксити, необхідно використати форматування, як я навів у прикладі - ОДИН відступ вліво і закриваючий тег на тому ж рядку
<img width="638" alt="Снимок экрана 2021-11-24 в 15 53 02" src="https://user-images.githubusercontent.com/66570142/143251529-8827e819-72bb-4910-b354-6ec946b02d07.png">